### ISSUE TITLE
Unable to install/enable the apigee_m10n_add_credit module in Drupal 8.9

### ISSUE THREAD

**Issue by micovery** [apigee/apigee-m10n-drupal#252]
## Description
Unable to install/enable the apigee_m10n_add_credit module in Drupal 8.9

## Steps to Reproduce

```
$ drush pm:list | grep -i "m10n"
  Apigee Kickstart                  Apigee Kickstart Monetization (Deprecated)                              Disabled                   
  Apigee                            Apigee Monetization (apigee_m10n)                                       Disabled   8.x-1.6         
  Apigee (Experimental)             Apigee Monetization Add Credit (apigee_m10n_add_credit)                 Disabled   8.x-1.6         
  Apigee (Experimental)             Apigee Monetization Teams (apigee_m10n_teams)                           Disabled   8.x-1.6         

$ drush en apigee_m10n
The following module(s) will be enabled: apigee_m10n, requirement

 Do you want to continue? (yes/no) [yes]:
 > yes

 [success] Successfully enabled: apigee_m10n, requirement


$ drush en apigee_m10n_teams
The following module(s) will be enabled: apigee_m10n_teams, apigee_edge_teams

 Do you want to continue? (yes/no) [yes]:
 > yes

 [success] Successfully enabled: apigee_m10n_teams, apigee_edge_teams
drupal@ourbank-dev-portal-8c7b87757-gkhzf:~/project$ drush en apigee_m10n_add_credit
The following module(s) will be enabled: apigee_m10n_add_credit, commerce, address, inline_entity_form, commerce_product, commerce_price, commerce_store, commerce_payment, commerce_order, commerce_number_pattern, profile, state_machine, commerce_cart, commerce_checkout

 Do you want to continue? (yes/no) [yes]:
 > yes

TypeError: Argument 2 passed to Drupal\jsonapi\Routing\Routes::Drupal\jsonapi\Routing\{closure}() must be an instance of Drupal\jsonapi\ResourceType\ResourceType, null given in /drupal/project/web/core/modules/jsonapi/src/Routing/Routes.php on line 434 #0 [internal function]: Drupal\jsonapi\Routing\Routes::Drupal\jsonapi\Routing\{closure}(false, NULL)
#1 /drupal/project/web/core/modules/jsonapi/src/Routing/Routes.php(436): array_reduce(Array, Object(Closure), false)
#2 /drupal/project/web/core/modules/jsonapi/src/Routing/Routes.php(339): Drupal\jsonapi\Routing\Routes::hasNonInternalTargetResourceTypes(Array)
#3 /drupal/project/web/core/modules/jsonapi/src/Routing/Routes.php(183): Drupal\jsonapi\Routing\Routes::getIndividualRoutesForResourceType(Object(Drupal\jsonapi_extras\ResourceType\ConfigurableResourceType))
#4 /drupal/project/web/core/modules/jsonapi/src/Routing/Routes.php(116): Drupal\jsonapi\Routing\Routes::getRoutesForResourceType(Object(Drupal\jsonapi_extras\ResourceType\ConfigurableResourceType), '/jsonapi')
#5 [internal function]: Drupal\jsonapi\Routing\Routes->routes()
#6 /drupal/project/web/core/lib/Drupal/Core/Routing/RouteBuilder.php(146): call_user_func(Array)
#7 /drupal/project/web/core/lib/Drupal/Core/ProxyClass/Routing/RouteBuilder.php(83): Drupal\Core\Routing\RouteBuilder->rebuild()
#8 /drupal/project/web/core/lib/Drupal/Core/Routing/RouteProviderLazyBuilder.php(68): Drupal\Core\ProxyClass\Routing\RouteBuilder->rebuild()
#9 /drupal/project/web/core/lib/Drupal/Core/Routing/RouteProviderLazyBuilder.php(99): Drupal\Core\Routing\RouteProviderLazyBuilder->getRouteProvider()
#10 /drupal/project/web/modules/contrib/admin_toolbar/admin_toolbar_tools/src/Plugin/Derivative/ExtraLinks.php(619): Drupal\Core\Routing\RouteProviderLazyBuilder->getRoutesByNames(Array)
#11 /drupal/project/web/modules/contrib/admin_toolbar/admin_toolbar_tools/src/Plugin/Derivative/ExtraLinks.php(109): Drupal\admin_toolbar_tools\Plugin\Derivative\ExtraLinks->routeExists('entity.block_co...')
#12 /drupal/project/web/core/lib/Drupal/Component/Plugin/Discovery/DerivativeDiscoveryDecorator.php(101): Drupal\admin_toolbar_tools\Plugin\Derivative\ExtraLinks->getDerivativeDefinitions(Array)
#13 /drupal/project/web/core/lib/Drupal/Component/Plugin/Discovery/DerivativeDiscoveryDecorator.php(87): Drupal\Component\Plugin\Discovery\DerivativeDiscoveryDecorator->getDerivatives(Array)
#14 /drupal/project/web/core/lib/Drupal/Core/Menu/MenuLinkManager.php(164): Drupal\Component\Plugin\Discovery\DerivativeDiscoveryDecorator->getDefinitions()
#15 /drupal/project/web/core/lib/Drupal/Core/Menu/MenuLinkManager.php(189): Drupal\Core\Menu\MenuLinkManager->getDefinitions()
#16 /drupal/project/web/modules/contrib/admin_toolbar/admin_toolbar_tools/admin_toolbar_tools.module(53): Drupal\Core\Menu\MenuLinkManager->rebuild()
#17 [internal function]: admin_toolbar_tools_entity_insert(Object(Drupal\commerce_product\Entity\ProductType), 'commerce_produc...')
#18 /drupal/project/web/core/lib/Drupal/Core/Extension/ModuleHandler.php(403): call_user_func_array('admin_toolbar_t...', Array)
#19 /drupal/project/web/core/lib/Drupal/Core/Config/Entity/ConfigEntityStorage.php(349): Drupal\Core\Extension\ModuleHandler->invokeAll('entity_insert', Array)
#20 /drupal/project/web/core/lib/Drupal/Core/Entity/EntityStorageBase.php(535): Drupal\Core\Config\Entity\ConfigEntityStorage->invokeHook('insert', Object(Drupal\commerce_product\Entity\ProductType))
#21 /drupal/project/web/core/lib/Drupal/Core/Entity/EntityStorageBase.php(460): Drupal\Core\Entity\EntityStorageBase->doPostSave(Object(Drupal\commerce_product\Entity\ProductType), false)
#22 /drupal/project/web/core/lib/Drupal/Core/Config/Entity/ConfigEntityStorage.php(263): Drupal\Core\Entity\EntityStorageBase->save(Object(Drupal\commerce_product\Entity\ProductType))
#23 /drupal/project/web/core/lib/Drupal/Core/Entity/EntityBase.php(395): Drupal\Core\Config\Entity\ConfigEntityStorage->save(Object(Drupal\commerce_product\Entity\ProductType))
#24 /drupal/project/web/core/lib/Drupal/Core/Config/Entity/ConfigEntityBase.php(616): Drupal\Core\Entity\EntityBase->save()
#25 /drupal/project/web/core/lib/Drupal/Core/Config/ConfigInstaller.php(371): Drupal\Core\Config\Entity\ConfigEntityBase->save()
#26 /drupal/project/web/core/lib/Drupal/Core/Config/ConfigInstaller.php(137): Drupal\Core\Config\ConfigInstaller->createConfiguration('', Array)
#27 /drupal/project/web/core/lib/Drupal/Core/ProxyClass/Config/ConfigInstaller.php(75): Drupal\Core\Config\ConfigInstaller->installDefaultConfig('module', 'commerce_produc...')
#28 /drupal/project/web/core/lib/Drupal/Core/Extension/ModuleInstaller.php(288): Drupal\Core\ProxyClass\Config\ConfigInstaller->installDefaultConfig('module', 'commerce_produc...')
#29 /drupal/project/web/core/lib/Drupal/Core/ProxyClass/Extension/ModuleInstaller.php(83): Drupal\Core\Extension\ModuleInstaller->install(Array, true)
#30 /drupal/project/vendor/drush/drush/src/Drupal/Commands/pm/PmCommands.php(78): Drupal\Core\ProxyClass\Extension\ModuleInstaller->install(Array, true)
#31 [internal function]: Drush\Drupal\Commands\pm\PmCommands->enable(Array, Array)
#32 /drupal/project/vendor/consolidation/annotated-command/src/CommandProcessor.php(257): call_user_func_array(Array, Array)
#33 /drupal/project/vendor/consolidation/annotated-command/src/CommandProcessor.php(212): Consolidation\AnnotatedCommand\CommandProcessor->runCommandCallback(Array, Object(Consolidation\AnnotatedCommand\CommandData))
#34 /drupal/project/vendor/consolidation/annotated-command/src/CommandProcessor.php(178): Consolidation\AnnotatedCommand\CommandProcessor->validateRunAndAlter(Array, Array, Object(Consolidation\AnnotatedCommand\CommandData))
#35 /drupal/project/vendor/consolidation/annotated-command/src/AnnotatedCommand.php(302): Consolidation\AnnotatedCommand\CommandProcessor->process(Object(Symfony\Component\Console\Output\ConsoleOutput), Array, Array, Object(Consolidation\AnnotatedCommand\CommandData))
#36 /drupal/project/vendor/symfony/console/Command/Command.php(255): Consolidation\AnnotatedCommand\AnnotatedCommand->execute(Object(Drush\Symfony\DrushArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#37 /drupal/project/vendor/symfony/console/Application.php(1005): Symfony\Component\Console\Command\Command->run(Object(Drush\Symfony\DrushArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#38 /drupal/project/vendor/symfony/console/Application.php(255): Symfony\Component\Console\Application->doRunCommand(Object(Consolidation\AnnotatedCommand\AnnotatedCommand), Object(Drush\Symfony\DrushArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#39 /drupal/project/vendor/symfony/console/Application.php(148): Symfony\Component\Console\Application->doRun(Object(Drush\Symfony\DrushArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#40 /drupal/project/vendor/drush/drush/src/Runtime/Runtime.php(118): Symfony\Component\Console\Application->run(Object(Drush\Symfony\DrushArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#41 /drupal/project/vendor/drush/drush/src/Runtime/Runtime.php(49): Drush\Runtime\Runtime->doRun(Array, Object(Symfony\Component\Console\Output\ConsoleOutput))
#42 /drupal/project/vendor/drush/drush/drush.php(72): Drush\Runtime\Runtime->run(Array)
#43 /drupal/project/vendor/drush/drush/drush(4): require('/drupal/project...')
#44 {main}
TypeError: Argument 2 passed to Drupal\jsonapi\Routing\Routes::Drupal\jsonapi\Routing\{closure}() must be an instance of Drupal\jsonapi\ResourceType\ResourceType, null given in Drupal\jsonapi\Routing\Routes::Drupal\jsonapi\Routing\{closure}() (line 434 of /drupal/project/web/core/modules/jsonapi/src/Routing/Routes.php).
PHP Fatal error:  Uncaught RuntimeException: Recursive router rebuild detected. in /drupal/project/web/core/lib/Drupal/Core/Routing/RouteBuilder.php:121
Stack trace:
#0 /drupal/project/web/core/lib/Drupal/Core/Routing/RouteBuilder.php(210): Drupal\Core\Routing\RouteBuilder->rebuild()
#1 /drupal/project/web/core/lib/Drupal/Core/Routing/RouteBuilder.php(222): Drupal\Core\Routing\RouteBuilder->rebuildIfNeeded()
#2 /drupal/project/web/core/lib/Drupal/Core/EventSubscriber/KernelDestructionSubscriber.php(51): Drupal\Core\Routing\RouteBuilder->destruct()
#3 [internal function]: Drupal\Core\EventSubscriber\KernelDestructionSubscriber->onKernelTerminate(Object(Symfony\Component\HttpKernel\Event\PostResponseEvent), 'kernel.terminat...', Object(Drupal\Component\EventDispatcher\ContainerAwareEventDispatcher))
#4 /drupal/project/web/core/lib/Drupal/Component/EventDispatcher/ContainerAwareEventDispatcher.php(111): call_user_func(Array, Object(Symfony\Component\HttpKernel\Event\PostResponseEvent), 'kernel.terminat...', Object(Drupal\Componen in /drupal/project/web/core/lib/Drupal/Core/Routing/RouteBuilder.php on line 121

```

## Actual Behavior
I get an error with a stack trace when trying to enable the `apigee_m10n_add_credit` module

## Expected Behavior
Enabling the apigee_m10n_add_credit should be successful 

## Version Info
```
$ drush status
 Drupal version   : 8.9.1                                        
 Site URI         : http://default                               
 DB driver        : mysql                                        
 DB hostname      : localhost                                    
 DB port          :                                              
 DB username      : drupal                                       
 DB name          : drupal                                       
 Database         : Connected                                    
 Drupal bootstrap : Successful                                   
 Default theme    : apigee_kickstart                             
 Admin theme      : seven                                        
 PHP binary       : /usr/bin/php7.2                              
 PHP config       : /etc/php/7.2/cli/php.ini                     
 PHP OS           : Linux                                        
 Drush script     : /drupal/project/vendor/drush/drush/drush     
 Drush version    : 9.7.2                                        
 Drush temp       : /tmp                                         
 Drush configs    : /drupal/project/vendor/drush/drush/drush.yml 
 Install profile  : apigee_devportal_kickstart                   
 Drupal root      : /drupal/project/web                          
 Site path        : sites/default                                
 Files, Public    : sites/default/files                          
 Files, Private   : /drupal/project/private                      
 Files, Temp      : /tmp                            
```

```
$ composer show
apigee/apigee-client-php                 2.0.5        Client library for connecting to the Apigee Edge API.
apigee/apigee_devportal_kickstart        1.15.0       A fast demo and starting point for Apigee Developer Portals.
asm89/stack-cors                         1.3.0        Cross-origin resource sharing library and stack middleware
behat/mink                               v1.8.1       Browser controller/emulator abstraction for PHP
behat/mink-browserkit-driver             v1.3.4       Symfony2 BrowserKit driver for Mink framework
behat/mink-goutte-driver                 v1.2.1       Goutte driver for Mink framework
behat/mink-selenium2-driver              v1.4.0       Selenium2 (WebDriver) driver for Mink framework
chi-teck/drupal-code-generator           1.32.0       Drupal code generator
clue/stream-filter                       v1.4.1       A simple and modern approach to stream filtering in PHP
commerceguys/addressing                  v1.0.8       Addressing library powered by CLDR and Google's address data.
commerceguys/intl                        v1.0.6       Internationalization library powered by CLDR data.
composer/ca-bundle                       1.2.7        Lets you find a path to the system CA bundle, and includes a fallback to the Mozilla CA bundle.
composer/composer                        1.10.7       Composer helps you declare, manage and install dependencies of PHP projects. It ensures you have ...
composer/installers                      v1.9.0       A multi-framework Composer library installer
composer/semver                          1.5.1        Semver library that offers utilities, version constraint parsing and validation.
composer/spdx-licenses                   1.5.3        SPDX licenses list and validation library.
composer/xdebug-handler                  1.4.2        Restarts a process without Xdebug.
consolidation/annotated-command          2.12.0       Initialize Symfony Console commands from annotated command class methods.
consolidation/config                     1.2.1        Provide configuration services for a commandline tool.
consolidation/filter-via-dot-access-data 1.0.0        This project uses dflydev/dot-access-data to provide simple output filtering for applications bui...
consolidation/log                        1.1.1        Improved Psr-3 / Psr\Log logger based on Symfony Console components.
consolidation/output-formatters          3.5.0        Format text by applying transformations provided by plug-in formatters.
consolidation/robo                       1.4.12       Modern task runner
consolidation/self-update                1.2.0        Provides a self:update command for Symfony Console applications.
consolidation/site-alias                 3.0.1        Manage alias records for local and remote sites.
consolidation/site-process               2.1.0        A thin wrapper around the Symfony Process Component that allows applications to use the Site Alia...
container-interop/container-interop      1.2.0        Promoting the interoperability of container objects (DIC, SL, etc.)
cweagans/composer-patches                1.6.7        Provides a way to patch Composer packages.
dflydev/dot-access-data                  v1.1.0       Given a deep data structure, access data by dot notation.
dnoegel/php-xdg-base-dir                 v0.1.1       implementation of xdg base directory specification for php
doctrine/annotations                     v1.4.0       Docblock Annotations Parser
doctrine/cache                           v1.6.2       Caching library offering an object-oriented API for many cache backends
doctrine/collections                     v1.4.0       Collections Abstraction library
doctrine/common                          v2.7.3       Common Library for Doctrine projects
doctrine/inflector                       v1.2.0       Common String Manipulations with regard to casing and singular/plural rules.
doctrine/instantiator                    1.3.1        A small, lightweight utility to instantiate objects in PHP without invoking their constructors
doctrine/lexer                           1.0.2        PHP Doctrine Lexer parser library that can be used in Top-Down, Recursive Descent Parsers.
drupal/address                           1.8.0        Provides functionality for storing, validating and displaying international postal addresses.
drupal/admin_toolbar                     2.3.0        Provides a drop-down menu interface to the core Drupal Toolbar.
drupal/adminimal_admin_toolbar           1.11.0       Adminimal styling brought to admin toolbar.
drupal/apigee_api_catalog                1.2.0        Apigee API Catalog for Drupal
drupal/apigee_edge                       1.10.0       Apigee Edge for Drupal.
drupal/apigee_m10n                       1.6.0        Apigee Edge Monetization for Drupal
drupal/better_exposed_filters            3.0.0-alpha6 Provides advanced options (such as links, checkboxes, or jQueryUI widgets) for exposed Views elem...
drupal/coder                             8.3.8        Coder is a library to review Drupal code.
drupal/commerce                          2.19.0       Drupal Commerce is a flexible eCommerce solution.
drupal/components                        1.1.0        Registers “component libraries” defined in modules and themes with the Twig system
drupal/core                              8.9.1        Drupal is an open source content management platform powering millions of websites and applications.
drupal/core-composer-scaffold            8.9.1        A flexible Composer project scaffold builder.
drupal/core-dev                          8.9.1        require-dev dependencies from drupal/drupal; use in addition to drupal/core-recommended to run te...
drupal/core-project-message              8.9.1        Adds a message after Composer installation.
drupal/core-recommended                  8.9.1        Locked core dependencies; require this project INSTEAD OF drupal/core.
drupal/ctools                            3.4.0        Provides a number of utility and helper APIs for Drupal developers and site builders.
drupal/default_content                   1.0.0-alpha9 Imports default content when a module is enabled
drupal/devel                             2.1.0        Various blocks, pages, and functions for developers.
drupal/email_registration                1.1.0        Allows users to register with an email address as their username.
drupal/entity                            1.0.0        Provides expanded entity APIs, which will be moved to Drupal core one day.
drupal/entity_reference_revisions        1.8.0        Entity Reference Revisions
drupal/file_link                         1.7.0        Extends the core Link module by storing metadata about the target file.
drupal/fontawesome                       2.16.0       The web's most popular icon set and toolkit.
drupal/inline_entity_form                1.0.0-rc6    Provides a widget for inline management (creation, modification, removal) of referenced entities.
drupal/jsonapi_extras                    3.14.0       JSON:API Extras provides a means to override and provide limited configurations to the default ze...
drupal/key                               1.14.0       Provides the ability to manage site-wide keys
drupal/paragraphs                        1.12.0       Enables the creation of Paragraphs entities.
drupal/pathauto                          1.8.0        Provides a mechanism for modules to automatically generate aliases for the content they manage.
drupal/profile                           1.1.0        Provides configurable user profiles.
drupal/radix                             4.8.0        Radix is a base theme for Drupal. It has Bootstrap, Sass, BrowserSync and Font Awesome built-in.
drupal/requirement                       1.0.0        A Drupal module to that gives modules a way to declare configuration requirement and suggestions.
drupal/state_machine                     1.0.0        Provides code-driven workflow functionality.
drupal/token                             1.7.0        Provides a user interface for the Token API, some missing core tokens.
drush/drush                              9.7.2        Drush is a command line shell and scripting interface for Drupal, a veritable Swiss Army knife de...
e0ipso/shaper                            1.2.3        Lightweight library to handle in and out transformations in PHP.
easyrdf/easyrdf                          0.9.1        EasyRdf is a PHP library designed to make it easy to consume and produce RDF.
egulias/email-validator                  2.1.17       A library for validating emails against several RFCs
fabpot/goutte                            v3.2.3       A simple PHP Web Scraper
fightbulc/moment                         1.32.1       Parse, validate, manipulate, and display dates in PHP w/ i18n support. Inspired by moment.js
firebase/php-jwt                         v5.2.0       A simple library to encode and decode JSON Web Tokens (JWT) in PHP. Should conform to the current...
grasmash/expander                        1.0.0        Expands internal property references in PHP arrays file.
grasmash/yaml-expander                   1.4.0        Expands internal property references in a yaml file.
guzzlehttp/guzzle                        6.5.4        Guzzle is a PHP HTTP client library
guzzlehttp/promises                      v1.3.1       Guzzle promises library
guzzlehttp/psr7                          1.6.1        PSR-7 message implementation that also provides common utility methods
instaclick/php-webdriver                 1.4.7        PHP WebDriver for Selenium 2
jcalderonzumba/gastonjs                  v1.2.0       PhantomJS API based server for webpage automation
jcalderonzumba/mink-phantomjs-driver     v0.3.3       PhantomJS driver for Mink framework
justinrainbow/json-schema                5.2.10       A library to validate a json schema.
laminas/laminas-diactoros                1.8.7p2      PSR HTTP Message implementations
laminas/laminas-escaper                  2.6.1        Securely and safely escape HTML, HTML attributes, JavaScript, CSS, and URLs
laminas/laminas-feed                     2.12.2       provides functionality for consuming RSS and Atom feeds
laminas/laminas-stdlib                   3.2.1        SPL extensions, array utilities, error handlers, and more
laminas/laminas-zendframework-bridge     1.0.4        Alias legacy ZF class names to Laminas Project equivalents.
league/container                         2.4.1        A fast and intuitive dependency injection container.
league/period                            3.4.0        A time range immutable value object
masterminds/html5                        2.3.0        An HTML5 parser and serializer.
mikey179/vfsstream                       v1.6.8       Virtual file system to mock the real file system in unit tests.
myclabs/deep-copy                        1.9.5        Create deep copies (clones) of your objects
nikic/php-parser                         v4.5.0       A PHP parser written in PHP
paragonie/random_compat                  v9.99.99     PHP 5.x polyfill for random_bytes() and random_int() from PHP 7
pear/archive_tar                         1.4.9        Tar file management class with compression support (gzip, bzip2, lzma2)
pear/console_getopt                      v1.4.3       More info available on: http://pear.php.net/package/Console_Getopt
pear/pear-core-minimal                   v1.10.10     Minimal set of PEAR core files to be used as composer dependency
pear/pear_exception                      v1.0.1       The PEAR Exception base class.
phar-io/manifest                         1.0.3        Component for reading phar.io manifest information from a PHP Archive (PHAR)
phar-io/version                          2.0.1        Library for handling version information and constraints
php-http/client-common                   1.10.0       Common HTTP Client implementations and tools for HTTPlug
php-http/discovery                       1.8.0        Finds installed HTTPlug implementations and PSR-7 message factories
php-http/guzzle6-adapter                 v1.1.1       Guzzle 6 HTTP Adapter
php-http/httplug                         v1.1.0       HTTPlug, the HTTP client abstraction for PHP
php-http/message                         1.8.0        HTTP Message related tools
php-http/message-factory                 v1.0.2       Factory interfaces for PSR-7 HTTP Message
php-http/promise                         v1.0.0       Promise used for asynchronous HTTP requests
phpdocumentor/reflection-common          2.1.0        Common reflection classes used by phpdocumentor to reflect the code structure
phpdocumentor/reflection-docblock        4.3.4        With this component, a library can provide support for annotations via DocBlocks or otherwise ret...
phpdocumentor/type-resolver              1.1.0        A PSR-5 based resolver of Class names, Types and Structural Element Names
phpspec/prophecy                         v1.10.3      Highly opinionated mocking framework for PHP 5.3+
phpunit/php-code-coverage                6.1.4        Library that provides collection, processing, and rendering functionality for PHP code coverage i...
phpunit/php-file-iterator                2.0.2        FilterIterator implementation that filters files based on a list of suffixes.
phpunit/php-text-template                1.2.1        Simple template engine.
phpunit/php-timer                        2.1.2        Utility class for timing
phpunit/php-token-stream                 3.1.1        Wrapper around PHP's tokenizer extension.
phpunit/phpunit                          7.5.20       The PHP Unit Testing framework.
psr/container                            1.0.0        Common Container Interface (PHP FIG PSR-11)
psr/http-message                         1.0.1        Common interface for HTTP messages
psr/log                                  1.1.3        Common interface for logging libraries
psy/psysh                                v0.10.4      An interactive shell for modern PHP.
ralouphie/getallheaders                  3.0.3        A polyfill for getallheaders.
sebastian/code-unit-reverse-lookup       1.0.1        Looks up which function or method a line of code belongs to
sebastian/comparator                     3.0.2        Provides the functionality to compare PHP values for equality
sebastian/diff                           3.0.2        Diff implementation
sebastian/environment                    4.2.3        Provides functionality to handle HHVM/PHP environments
sebastian/exporter                       3.1.2        Provides the functionality to export PHP variables for visualization
sebastian/global-state                   2.0.0        Snapshotting of global state
sebastian/object-enumerator              3.0.3        Traverses array structures and object graphs to enumerate all referenced objects
sebastian/object-reflector               1.1.1        Allows reflection of object attributes, including inherited and non-public ones
sebastian/recursion-context              3.0.0        Provides functionality to recursively process PHP variables
sebastian/resource-operations            2.0.1        Provides a list of PHP built-in functions that operate on resources
sebastian/version                        2.0.1        Library that helps with managing the version number of Git-hosted PHP projects
seld/jsonlint                            1.8.0        JSON Linter
seld/phar-utils                          1.1.0        PHAR file format utilities, for when PHP phars you up
squizlabs/php_codesniffer                3.5.5        PHP_CodeSniffer tokenizes PHP, JavaScript and CSS files and detects violations of a defined set o...
stack/builder                            v1.0.5       Builder for stack middlewares based on HttpKernelInterface.
symfony-cmf/routing                      1.4.1        Extends the Symfony2 routing component for dynamic routes and chaining several routers
symfony/browser-kit                      v3.4.42      Symfony BrowserKit Component
symfony/class-loader                     v3.4.41      Symfony ClassLoader Component
symfony/console                          v3.4.41      Symfony Console Component
symfony/css-selector                     v3.4.42      Symfony CssSelector Component
symfony/debug                            v3.4.41      Symfony Debug Component
symfony/dependency-injection             v3.4.41      Symfony DependencyInjection Component
symfony/deprecation-contracts            v2.1.2       A generic function and convention to trigger deprecation notices
symfony/dom-crawler                      v3.4.42      Symfony DomCrawler Component
symfony/event-dispatcher                 v3.4.41      Symfony EventDispatcher Component
symfony/filesystem                       v3.4.42      Symfony Filesystem Component
symfony/finder                           v3.4.42      Symfony Finder Component
symfony/http-foundation                  v3.4.41      Symfony HttpFoundation Component
symfony/http-kernel                      v3.4.41      Symfony HttpKernel Component
symfony/inflector                        v5.1.2       Symfony Inflector Component
symfony/lock                             v3.4.42      Symfony Lock Component
symfony/options-resolver                 v4.4.10      Symfony OptionsResolver Component
symfony/phpunit-bridge                   v3.4.42      Symfony PHPUnit Bridge
symfony/polyfill-ctype                   v1.17.0      Symfony polyfill for ctype functions
symfony/polyfill-iconv                   v1.17.0      Symfony polyfill for the Iconv extension
symfony/polyfill-intl-grapheme           v1.17.0      Symfony polyfill for intl's grapheme_* functions
symfony/polyfill-intl-idn                v1.17.0      Symfony polyfill for intl's idn_to_ascii and idn_to_utf8 functions
symfony/polyfill-intl-normalizer         v1.17.0      Symfony polyfill for intl's Normalizer class and related functions
symfony/polyfill-mbstring                v1.17.0      Symfony polyfill for the Mbstring extension
symfony/polyfill-php56                   v1.17.0      Symfony polyfill backporting some PHP 5.6+ features to lower PHP versions
symfony/polyfill-php70                   v1.17.0      Symfony polyfill backporting some PHP 7.0+ features to lower PHP versions
symfony/polyfill-php72                   v1.17.0      Symfony polyfill backporting some PHP 7.2+ features to lower PHP versions
symfony/polyfill-php80                   v1.17.0      Symfony polyfill backporting some PHP 8.0+ features to lower PHP versions
symfony/polyfill-util                    v1.17.0      Symfony utilities for portability of PHP codes
symfony/process                          v3.4.41      Symfony Process Component
symfony/property-access                  v4.4.10      Symfony PropertyAccess Component
symfony/property-info                    v4.4.10      Symfony Property Info Component
symfony/psr-http-message-bridge          v1.1.2       PSR HTTP message bridge
symfony/routing                          v3.4.41      Symfony Routing Component
symfony/serializer                       v3.4.41      Symfony Serializer Component
symfony/string                           v5.1.2       Symfony String component
symfony/translation                      v3.4.41      Symfony Translation Component
symfony/validator                        v3.4.41      Symfony Validator Component
symfony/var-dumper                       v4.4.10      Symfony mechanism for exploring and dumping PHP variables
symfony/yaml                             v3.4.41      Symfony Yaml Component
theseer/tokenizer                        1.1.3        A small library for converting tokenized PHP source code into XML and potentially other formats
twig/twig                                v1.42.5      Twig, the flexible, fast, and secure template language for PHP
typo3/phar-stream-wrapper                v3.1.4       Interceptors for PHP's native phar:// stream handling
webflo/drupal-finder                     1.2.0        Helper class to locate a Drupal installation from a given path.
webmozart/assert                         1.9.0        Assertions to validate method input/output with nice error messages.
webmozart/path-util                      2.3.0        A robust cross-platform utility for normalizing, comparing and modifying file paths.
```

**Comment by micovery** [comment]
If I re-run the drush command to enable the module multiple times, at some point it stops giving me the stack trace, and instead it gives me this error:

```
$ drush en apigee_m10n_add_credit
The following module(s) will be enabled: apigee_m10n_add_credit, commerce_payment, commerce_cart, commerce_checkout

 Do you want to continue? (yes/no) [yes]:
 > yes


In UnmetDependenciesException.php line 98:
                                                                                                                                                           
  Configuration objects provided by <em class="placeholder">commerce_checkout</em> have unmet dependencies: <em class="placeholder">core.entity_view_disp  
  lay.commerce_product_variation.default.summary (commerce_product.commerce_product_variation_type.default)</em>   

```

**Comment by micovery** [comment]
Trying the same process using the UI instead of drush, I go through a similar experience.

I try to enable the module a few times, and it just errors out to a brank page. When I inspect the error log I see:

![image](https://user-images.githubusercontent.com/1885394/95512680-0c290700-096e-11eb-97d4-f71b2069bea5.png)

which is the same stack trace I get from drush.

I keep trying to install the module, and it looks like each time it makes some progress. Each time it shows less dependencies in the confirmation window. Eventually at some point I get to this point:

![image](https://user-images.githubusercontent.com/1885394/95512901-5c07ce00-096e-11eb-9b5a-793272268001.png)

where it only shows the "Commerce Checkout" module as a dependency, but when I hit continue, it errors out with the following:

![image](https://user-images.githubusercontent.com/1885394/95512530-cc621f80-096d-11eb-9123-6878e9311199.png)

**Comment by arlina-espinoza** [comment]
Thanks for the report @micovery. I'm starting to look into this, and it seems it has to do with the core `jsonapi` module being enabled on the site. If jsonapi is disabled, installation seems to proceed without errors. I'll see what else I can find out.

**Comment by micovery** [comment]
@arlina-espinoza thanks for the very quick response! You are right. As a work-around I removed the following two modules:

1. JSON:API_extras
2. JSON:API

After doing that I was able to successfully install the  `apigee_m10n_add_credit` module.

![image](https://user-images.githubusercontent.com/1885394/95519084-41872200-0979-11eb-846e-6b4c26b8bcf5.png)

**Comment by arlina-espinoza** [comment]
Good to hear @micovery !

In case someone else finds it useful, the error has been reported to Drupal core JSON API module: https://www.drupal.org/project/drupal/issues/2996114#comment-13332101

> So it seems like this bug could occur if routes are rebuilt before config sync is finished and bundles are installed completely.

Patch [#124](https://www.drupal.org/project/drupal/issues/2996114#comment-13460297) provides a fix for Drupal 8.9.x.

I'll close this issue, but feel free to reopen if you need more info.
